suricata.yaml.in

```diff
@@ -350,6 +350,7 @@ outputs:
       force-magic: no   # force logging magic on all stored files
       force-md5: no     # force logging of md5 checksums
       force-filestore: no # force storing of all files
+      size: 1mb         # reassemble 1mb into a stream, set to no to disable
```

Comment:
why is this not named stream-depth like in the app layer? The 'size' name is confusing as it suggests it's about the file size, which it isn't.